github_rules/github_org_auth_modified.py

```diff
@@ -0,0 +1,19 @@
+def rule(event):
+    auth_change_events = [
+        "org.saml_disabled",
+        "org.saml_enabled",
+        "org.disable_two_factor_requirement",
+        "org.enable_two_factor_requirement",
+        "org.update_saml_provider_settings",
+        "org.enable_oauth_app_restrictions",
+        "org.disable_oauth_app_restrictions",
+    ]
```

Comment:
[Optional] Typically we define lists like these outside of the scope of `rule`, so that they're instantiated once per lambda invocation and not once per rule invocation. Technically there should be a small speed increase that way.

I suppose the counterargument is that this way saves some memory for other rules to run, so it really just depends on what we're optimizing for. Since we don't have an official standard I'm ok with this as is, just pointing out is more "standard" to have static lists defined outside the scope of `rule`.

Reply:
I think once per Lambda makes sense, and I'm down with going with the standard.  My default mode is to not have things defined outside of scopes in which they are used but saving instantiation cycles is logical here